core/feconf.py

```diff
@@ -26,7 +26,8 @@
 from core.constants import constants
 
 from typing import Dict, List, Union
-from typing_extensions import TypedDict
+
+from typing_extensions import TypedDict # isort:skip  # pylint: disable=wrong-import-position
```

Comment:
For most other files in the codebase that import typing_extensions, I don't see these pragmas being used (isort: skip, etc.) Why are they needed here?

Reply:
Its not needed, i introduce it before for third party import and I forgot to remove it.
Its removed PTAL